src/sonic-py-common/sonic_py_common/general.py

```diff
@@ -43,7 +44,7 @@ def getstatusoutput_noshell(cmd):
     return exitcode, output
 
 
-def getstatusoutput_noshell_pipe(cmd0, *args):
+def getstatusoutput_noshell_pipe(cmd0, *args, return_cmd=True):
```

Comment:
>return_cmd [](http://example.com/codeflow?start=46&length=10)

Possible to fix the bug without new parameter?

Reply:
The user can choose to display or not to display the output. Without new parameter, the program forces to display the output?